web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.ts

```diff
@@ -113,7 +113,7 @@ export default function useAccessRequestCheckout() {
           resourceIds: data
             .filter(d => d.kind !== 'role')
             .map(d => ({
-              name: d.name,
+              name: d.id,
```

Comment:
Is it something that's going to be partially addressed by your other two access request PRs? I can see how one can make this mistake as the field names are a bit confusing.

If it's not going to be addressed by your PRs, could you add a comment explaining why we have to use id over name?

Reply:
I didn't change this part in my PRs (although, now we have a foundation to make it better). I will leave a comment.